### PR TITLE
tests: Increase sleep in test_pidfile.sh

### PR DESCRIPTION
**Pull request by dzafman**

### CODE REVIEW

**Comment by dzafman** [comment]
retest this please 

Test failures: 3 - test_objectstore_memstore.sh (Failed)

File: src/test/test_pidfile.sh [hunk 1]
```diff
@@ -50,11 +50,11 @@ function TEST_pidfile() {
 
     # no daemon can use a pidfile that is owned by another daemon
     run_mon $dir a || return 1
-    sleep 2
+    sleep 5
```
Comment (amitkumar50) [inline src/test/test_pidfile.sh]: @dzafman What's reference of Increase from `2` to `5`?

Comment (dzafman) [inline src/test/test_pidfile.sh]: We saw this test fail, again.  We are starting a background daemon then trying to start another instance.  The first invocation must make enough progress for it to lock its pid file.

**Comment by dzafman** [comment]
retest this please

**Pull request merged**